release/charts/chart_management.go

```diff
@@ -43,7 +48,12 @@ func ChartArgs(ctx context.Context, c *config.ChartsRelease) ([]string, error) {
 
 // VersionArgs will return the list of available versions for the target chart
 func VersionArgs(ctx context.Context, c *config.ChartsRelease, ch string) ([]string, error) {
-	status, err := loadState(filepath.Join(c.Workspace, "state.json"))
+	var (
+		status *status
```

Comment:
Almost all of the variable declarations in this PR are unnecessary and should be assigned with the `:=` call.